src/main/java/com/epam/selectioncommittee/controller/FacultyController.java

```diff
@@ -0,0 +1,43 @@
+package com.epam.selectioncommittee.controller;
+
+import com.epam.selectioncommittee.dto.FacultyDto;
+import com.epam.selectioncommittee.service.FacultyService;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@AllArgsConstructor
+@Slf4j
+public class FacultyController {
+    private final FacultyService facultyService;
+
+    @ResponseStatus(HttpStatus.OK)
+    @GetMapping(value = "/faculty")

```

Comment:
I would recommend to use @RequestMapping("/faculty") instead of duplicate the same url in each method.
Also check the other controller for the same issue